diff --git a/common/dynamic/service_contexts b/common/dynamic/service_contexts
index 5f6ca09..394f3e4 100644
--- a/common/dynamic/service_contexts
+++ b/common/dynamic/service_contexts
@@ -1,2 +1,3 @@
 vendor.lineage.powershare.IPowerShare/default        u:object_r:hal_lineage_powershare_service:s0
 vendor.lineage.health.IChargingControl/default        u:object_r:hal_lineage_health_service:s0
+vendor.lineage.health.IFastCharge/default             u:object_r:hal_lineage_health_service:s0
